src/services/completions.ts

```diff
@@ -267,68 +267,87 @@ namespace ts.Completions {
             return undefined;
         }
 
-        if (node.parent.kind === SyntaxKind.PropertyAssignment &&

```

Comment:
Same here. Can you revert the if&rarr;switch refactoring to make it obvious what actually changed here? I'm happy to review it separately, but it makes reading that actual changes painful.